<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="152883" started-at="2021-05-06T13:53:23Z" finished-at="2021-05-06T13:55:56Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="152883" started-at="2021-05-06T13:53:23Z" finished-at="2021-05-06T13:55:56Z">
      <class name="crmframework.crmAutomation.ReferenceDataPageTest">
        <test-method status="PASS" signature="initialize()[pri:0, instance:crmframework.crmAutomation.ReferenceDataPageTest@28d25987]" name="initialize" is-config="true" duration-ms="3632" started-at="2021-05-06T19:23:23Z" finished-at="2021-05-06T19:23:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="TS001_VerifyHomePageTest()[pri:0, instance:crmframework.crmAutomation.ReferenceDataPageTest@28d25987]" name="TS001_VerifyHomePageTest" duration-ms="112787" started-at="2021-05-06T19:23:27Z" finished-at="2021-05-06T19:25:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TS001_VerifyHomePageTest -->
        <test-method status="FAIL" signature="TS009_VerifySearchChannelsTest()[pri:0, instance:crmframework.crmAutomation.ReferenceDataPageTest@28d25987]" name="TS009_VerifySearchChannelsTest" duration-ms="36441" started-at="2021-05-06T19:25:20Z" finished-at="2021-05-06T19:25:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at crmframework.crmAutomation.ReferenceDataPageTest.TS009_VerifySearchChannelsTest(ReferenceDataPageTest.java:762)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TS009_VerifySearchChannelsTest -->
      </class> <!-- crmframework.crmAutomation.ReferenceDataPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
